packages/ics721/src/execute.rs

```diff
@@ -80,7 +99,7 @@ where
         sender: String,
         msg: Binary,
     ) -> Result<Response<T>, ContractError> {
-        if PROXY.load(deps.storage)?.is_some() {
+        if OUTGOING_PROXY.load(deps.storage)?.is_some() {
```

Comment:
Remember a long time ago when I said we should name it outgoing_proxy and incoming_proxy? Well, there you go its necessary now.

Reply:
Yes, that was in context of proxy contracts, there I renamed it to outgoing proxy. Didnt dare to change it in ics721 as well. Due to new incoming proxy though - there is a need now to distinct incoming from outgoing proxy.

And yeah - as in most cases: you r right ;)!